### ISSUE TITLE
[Feature request] Training code for LLaVA-NeXT-34B

### ISSUE THREAD

**Issue by zhly0** [haotian-liu/LLaVA#1174]
### feature

Hi，is the training code for LLaVA-NeXT-34B will release in February or March？

**Comment by awzhgw** [comment]
+1

**Comment by Reichenbachian** [comment]
+1

**Comment by loveunk** [comment]
Same question

**Comment by awzhgw** [comment]
Same question

**Comment by yushuinanrong** [comment]
+1

**Comment by jasonaidm** [comment]
+ Same question

**Comment by chricro** [comment]
Thank you for your efforts, @haotian-liu. Can you please share any progress updates regarding the pretraining code release for the 1.6 series? cheers

**Comment by hunterheiden** [comment]
Yes, it would be fantastic to hear if we can expect to see training code before the end of June or if we should expect it later on in the year! Very eager to build on top of this awesome work
